Python/susankoju/task2/read_sqlite.py

```diff
@@ -0,0 +1,6 @@
+import sqlite3
+
+conn = sqlite3.connect('output/results.db')
+cursor = conn.cursor()
```

Comment:
You can use context manager here
with con:
con.execute(
Reference:
https://docs.python.org/3/library/sqlite3.html#using-the-connection-as-a-context-manager